airflow/operators/bash_operator.py

```diff
@@ -87,14 +88,14 @@ def execute(self, context):
 
                 self.sp = sp
 
-                logging.info("Output:")
-                line = ''
+                output = StringIO()
                 for line in iter(sp.stdout.readline, b''):
-                    line = line.decode(self.output_encoding).strip()
-                    logging.info(line)
+                    output.write(line)
+                output = output.getvalue()
```

Comment:
This loads the entire log into memory. We might need to find a better way to do this.

We have Spark jobs that sometimes get a little too verbose and have very large logs. This change would definitely not work there.